frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices-mode/install.tsx

```diff
@@ -93,6 +95,10 @@ export const CreateAttachedDevicesCluster: React.FC<CreateAttachedDevicesCluster
     </Alert>
   ) : (
     <CreateStorageClusterWizard
+      getStep={getStep}
+      getParamString={getParamString}
+      getIndex={getIndex}
+      getAnchor={getAnchor}
```

Comment:
why not passing `navUtils` from here as well?

Reply:
My bad, fixed.